pkg/edgedictionaryitem/describe.go

```diff
@@ -0,0 +1,49 @@
+package edgedictionaryitem
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v2/fastly"
+)
+
+// DescribeCommand calls the Fastly API to describe a service.
+type DescribeCommand struct {
+	common.Base
+	manifest manifest.Data
+	Input    fastly.GetDictionaryItemInput
+}
+
+// NewDescribeCommand returns a usable command registered under the parent.
+func NewDescribeCommand(parent common.Registerer, globals *config.Data) *DescribeCommand {
+	var c DescribeCommand
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("describe", "Show detailed information about a Fastly edge dictionary item").Alias("get")
+	c.CmdClause.Flag("service-id", "Service ID").Short('s').StringVar(&c.manifest.Flag.ServiceID)
+	c.CmdClause.Flag("name", "Name of dictionary").Short('n').Required().StringVar(&c.Input.DictionaryID)
+	c.CmdClause.Flag("key", "Dictionary item key").Required().StringVar(&c.Input.ItemKey)
```

Comment:
Is there any reason we don't provide a `-k` short key like we do for `--name` with `-n`? 

If there's no reason not to, then could you just go back and ensure we have `.Short()` equivalents wherever necessary for consistency.

Reply:
Yes. I didn't use a `-k` for `--key` because that would suggest `-v` for `--value`, which is already taken by `--verbose` which is a global option.